### DISCUSSION TITLE
Handling integration of multiome replicates?

### DISCUSSION THREAD

**Discussion by ahepperla** [satijalab/seurat#4038] [Q&A]
Hi,

I'm working through the vignette on WNN and 10X multiome data. My dataset consists of two conditions of two replicates each (ie ATAC1+ATAC2+RNA1+RNA2 for a condition), and I'd like to perform integration of the data to correct for collection batch effects. Would it be best to run integration of each assay across the suerat objects or is there a better way to perform integration that includes both the ATAC and RNA data all at once?

Thanks.

**Comment by arvand1992** [comment]
Hi, 
Did you happen to figure out the best way to do the integration? I have the same issue and wondering how to proceed. 

Thanks,
